.github/workflows/coding-standards.yml

```
@@ -0,0 +1,48 @@
+name: "Coding Standards"
+
+on:
+  pull_request:
+    paths:
+      - 'source/**/*.php'
```

Add the .github directory so the GHA is run for this PR.
```suggestion
      - '.github/workflows/*.yml'
```